Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrBridgeBuilder.kt

```diff
@@ -16,6 +19,44 @@ class BridgeBuilderResult(
         val excludedStubs: Set<StubIrElement>
 )
 
+private data class CCalleeWrapper(val name: String, val lines: List<String>)
+
+/**
+ * Some functions doesn't have an address (e.g. macros-based or builtins).
+ * To solve this problem we generate a wrapper function.
```

Comment:
Can we generate wrappers only for those functions?

Reply:
It's hard to distinguish functions that require wrapping from those that doesn't.

Reply:
The calls to wrapper are actually indirect (through the generated const global). I hope that optimizer is smart enough to make it direct.
Also we can try to avoid using globals in some cases. Globals are mostly required because generally there is no reliable way to refer the function from stubs directly (e.g. it may be C `static`). But we can reliably refer the wrapper when it is present. This may although complicate the semantics of `CCall` (make it hold the name of global for variadic functions and the name of the function for the rest).